### ISSUE TITLE
Admin Settings Panel - Button to open working hours modal is not keyboard operable

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#233]
The button required to open the Company Working Hours settings modal is not reachable or operable by keyboard.

<img width="910" alt="Screen Shot 2020-04-20 at 8 51 28 AM" src="https://user-images.githubusercontent.com/31868510/79753844-8f816e00-82e4-11ea-91fc-5a2eda6a9ff7.png">

You can read about this success criterion here:

https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html
